lib/models/Quote.js

```diff
@@ -16,7 +16,12 @@ class Quote {
   }
 
   static async insert() {
-    // implement insert to add new quote
+    const { rows } =
+      await pool.query(`INSERT INTO quotes (episode_id, character_id, detail)
```

Comment:
To get this method to be working to post any requested quote it needs to take in a character_id, episode_id, and detail as parameters. Then, to sanitize the input and prevent SQL injection, the values turn into ($1, $2, $3), and we can add an array with the parameters we want to replace those placeholder values with (our original params)